bindings/java/src/org/sleuthkit/datamodel/Content.java

```diff
@@ -195,6 +195,21 @@
 	 */
 	public AnalysisResultAdded newAnalysisResult(BlackboardArtifact.Type artifactType, Score score, String conclusion, String configuration, String justification, Collection<BlackboardAttribute> attributesList) throws TskCoreException;
 
+	/**
+	 * Create and add a data artifact associated with this content.
+	 *
+	 * @param artifactType   Type of analysis result artifact to create.
+	 * @param attributesList Additional attributes to attach to this data
+	 *                       artifact.
+	 * @param osAccount      The OS account associated with the artifact. May be
+	 *                       null.
+	 *
+	 * @return DataArtifact New data artifact.
+	 *
+	 * @throws TskCoreException If a critical error occurred within tsk core.
+	 */
+	public DataArtifact newDataArtifact(BlackboardArtifact.Type artifactType, Collection<BlackboardAttribute> attributesList, OsAccount osAccount) throws TskCoreException;
+	
```

Comment:
@bcarrier, @jayaramcs and @raman-bt....

Given the move to DTOs, perhaps we do not want to add this convenience API that requires calls through SleuthkitCase?

Reply:
I think Content already has several of these kinds of methods and will need a reference to SleuthkitCase already. I don't see how Content could go to a pure DTO at this point.